DataFormats/MemoryResources/include/MemoryResources/MemoryResources.h

```diff
@@ -108,11 +108,11 @@ class ChannelResource : public FairMQMemoryResource
 
   bool do_is_equal(const boost::container::pmr::memory_resource& other) const noexcept override
   {
-    const FairMQMemoryResource* that = dynamic_cast<const FairMQMemoryResource*>(&other);
-    if (that && that->getTransportFactory() == factory) {
-      return true;
-    }
-    return false;
+    //const FairMQMemoryResource* that = dynamic_cast<const FairMQMemoryResource*>(&other);
```

Comment:
Can you remove the commented lines? They'll still be available in the git history.

Reply:
ok, done